sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/ImageTests.cs

```diff
@@ -48,7 +48,7 @@ public void TestCreateImage_without_DiskEncryptionSet()
             {
                 using (MockContext context = MockContext.Start(this.GetType()))
                 {
-                    Environment.SetEnvironmentVariable("AZURE_VM_TEST_LOCATION", "FranceCentral");
+                    Environment.SetEnvironmentVariable("AZURE_VM_TEST_LOCATION", "eastus2");
```

Comment:
same comment as above

Reply:
@markcowl , agreed, we were not in charge of these tests and don't currently have the knowledge to change it. The DiskRP team needs to modify this. We will be curating these tests in the future.